package.json

```diff
@@ -129,9 +129,11 @@
 		"@fluidframework/test-tools": "^0.2.3074",
 		"@microsoft/api-documenter": "^7.17.9",
 		"@microsoft/api-extractor": "^7.22.2",
+		"@octokit/core": "^4.0.5",
 		"c8": "^7.7.1",
 		"concurrently": "^6.2.0",
 		"copyfiles": "^2.4.1",
+		"danger": "^10.9.0",
```

Comment:
Since we can't include a comment in here, you should note that this is a workaround in the pPR description and include a TODO comment about it in build-tools/packages/build-cli/src/commands/run/bundleStats.ts